docs/hero/captions/src/HeroStitched.tsx

```
@@ -0,0 +1,109 @@
+    <>
+      {captions.map((cap, ci) => {
+        const phrases = PHRASES[cap.group] || [];
+        const from = Math.round(cap.atSec * fps);
+        const nextAt = captions[ci + 1] ? Math.round(captions[ci + 1].atSec * fps) : clipFrames;
+        const shotFrames = Math.max(1, nextAt - from);
+        return (
+          <Sequence key={cap.group} from={from} durationInFrames={shotFrames} name={cap.group}>
+            {phrases.map((ph, i) => {
+              const { x, y } = chipPlacement(i, phrases.length);
+              return (
+                <CaptionChip key={i} text={ph} delayFrames={Math.round(i * STAGGER_SEC * fps)} shotFrames={shotFrames} x={x} y={y} />
```

Biome (2.5.1)

[error] Avoid using the index of an array as key property in an element.

(lint/suspicious/noArrayIndexKey)